src/components/Api.js

```diff
@@ -0,0 +1,88 @@
+//const config = {}; //i don't really see that as a necessity at this point
+
+export default class Api {
+  constructor() {
+    this._baseUrl = "https://around-api.en.tripleten-services.com/v1";
+    this._headers = {
+      authorization: "5bd639c7-c1c3-496d-947f-3773995c6d1b",
+      "Content-Type": "application/json",
+    };
```

Comment:
According to the task you need to pass full `headers` and `baseUrl`into the constructor of `Api`. See here https://disk.yandex.ru/i/J93dbJp7eypXKQ . You shouldn't hardcode them inside `Api` to be able to create different api instances with different data in the future